docs/source/techniques/SANS-Toml-v1.rst

```diff
@@ -1152,128 +1820,247 @@ in :ref:`algm-TOFSANSResolutionByPixel`.
-    w1 = x
-    w2 = x
+    h1 = x1
+    w1 = y1
+    h2 = x2
+    w2 = y2
 
 **Existing Example:**
 
 ..  code-block:: none
 
     QRESOL/H1=16.0
-    QRESOL/H2=8.0
     QRESOL/W1=16.0
+    QRESOL/H2=8.0
     QRESOL/W2=8.0
 
-**Existing Replacement**
+**Replacement Example**
 
 ..  code-block:: none
 
   [q_resolution]
-    h1 = 16.0
-    h2 = 8.0
-    w1 = 16.0
-    w2 = 8.0
+    h1 = 0.016
+    w1 = 0.016
+    h2 = 0.008
+    w2 = 0.008
+
+.. ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
+.. David you had these as mm even in TOML; I assume that was a mistake?
+.. ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
+
+QRESOL/LCOLLIM=z
+----------------
+
+This command was used to specify the length of the collimation - the distance
+between the source and sample apertures/slits/jaws - to be used in the
+estimation of the Q-resolution.
+
+For more information, see the
+`TOFSANSResolutionByPixel <https://docs.mantidproject.org/nightly/algorithms/TOFSANSResolutionByPixel-v1.html>`_
+algorithm description.
+
+Note that because the collimation length is frequently altered, the ISIS
+SANS Group decided to place the TOML replacement in the [instrument.configuration]
+block at the top of TOML User Files instead of the [q_resolution] block.
 
-SAMPLE/OFFSET x
+Also note that the collimation length was historically specified in metres too.
+
+**Replacement**
+
+..  code-block:: none
+
+  [instrument.configuration]
+    collimation_length = z
+
+**Existing Example:**
+
+..  code-block:: none
+
+    QRESOL/LCOLLIM=4.0
+
+**Replacement Example**
+
+..  code-block:: none
+
+  [instrument.configuration]
+    collimation_length = 4.0
+
+SAMPLE/OFFSET z
 ---------------
 
+This command was used to specify any correction to the default Z coordinate
+in the Mantid Instrument Definition File defining the nominal position of the
+sample. The offset is a relative value with positive offsets translating the
+sample position *towards* the detector(s).
+
 **Replacement**
 
 ..  code-block:: none
 
   [instrument.configuration]
-    sample_offset = n
+    sample_offset = z
 
 **Existing Example:**
 
 ..  code-block:: none
 
     SAMPLE/OFFSET -60
 
-**Existing Replacement**
+**Replacement Example**
 
 ..  code-block:: none
 
   [instrument.configuration]
     sample_offset = -0.06
 
+SAMPLE/PATH[/ON][/OFF]
+----------------------
+
+This command was used to activate a correction to calculated transmissions
+arising from the longer path length through the sample/can at high angles.
+Unlike MON/HABPATH[/ON][/OFF] this command was generic.
+
+For more information, see the
+`SANSWideAngleCorrection <https://docs.mantidproject.org/nightly/algorithms/SANSWideAngleCorrection-v1.html>`_
+algorithm description.
 
-set centre a b
+**Existing Example:**
+
+..  code-block:: none
+
+    SAMPLE/PATH/ON
+
+**Replacement Example**
+
+Unsupported
+
+.. ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
```

Comment:
In this case, it's supported in the existing user files but not TOML as we haven't selected a name / location for this.

@smk78 If you could send me a user file which uses it (to write a test to make sure we get the same result from the old / new files), and we could decide what to call it / where to put it?
Internally it uses an attribute called `wide_angle_correction`, so maybe this would be a good name?

Reply:
Ok, I'll need to investigate.

Reply:
So...

The (now unsupported) legacy commands MON/HABEFF & MON/HABPATH were in routine use on LOQ upto ~start of 2011. This date approximately corresponds to the point at which LOQ data reduction moved over from COLETTE to Mantid. The new Mantid User Files then noted:

```
PRINT MON/HABEFF correction not yet implemented
PRINT MON/HABPATH correction not yet implemented
```

It looks to me as if the (supported) legacy command SAMPLE/PATH was developed at SNS and that ISIS was unaware of it, because I cannot find an instance of it being used in a LOQ User File (and LOQ would be the instrument that would benefit most, followed by SANS2D). Thus it would be good to have this command TOML-ised, please. Maybe something like:

```
[transmission]
wide_angle_correction = true | false
```

Reply:
Some observations from testing SAMPLE/PATH:

- we will need a field added to the SANS Interface to show if it is enabled or not (in Settings > Adjustment > Transmission is probably the most appropriate place)
- if the can transmission being corrected is ~unity - as can often happen if we just use the instrument background as the can and so Can Transmission and the Can Direct are both the Direct Beam - and the front/hab detector is selected then the reduced intensities and their uncertainties can get output as -Nan. Bit odd. The rear/main detector data reduce fine.
- The transmission workspaces that are written (or the values to file) appear to be identical whether or not SAMPLE/PATH is ON or not. I assume this means that the correction is applied internally after these values are generated. It would be good to be able to see the corrected values.
- This correction, unsurprisingly, has negligible impact on rear/main detector data:
![image](https://user-images.githubusercontent.com/10938679/146806016-187f360d-84ae-4d33-97c6-3d8725524b0b.png)
but it does have a small effect on the front/hab data
![image](https://user-images.githubusercontent.com/10938679/146806102-6cdbbaad-142a-4480-8fcb-f3dcac9df778.png)
Note that a different sample, for example, one that is more absorbing than this sample, will likely show a bigger effect.

[USER_LOQ_211G_M3_STEVE_TESTING_TOML__M3_main_log__DONT_USE_With_SamplePath.txt](https://github.com/mantidproject/mantid/files/7746932/USER_LOQ_211G_M3_STEVE_TESTING_TOML__M3_main_log__DONT_USE_With_SamplePath.txt)
[USER_LOQ_211G_M3_STEVE_TESTING_TOML__M3_hab_log__DONT_USE_With_SamplePath.txt](https://github.com/mantidproject/mantid/files/7746934/USER_LOQ_211G_M3_STEVE_TESTING_TOML__M3_hab_log__DONT_USE_With_SamplePath.txt)
Data are from LOQ Cycle 20/2
![image](https://user-images.githubusercontent.com/10938679/146806595-a1374fa6-45f9-426c-9574-cea2b985d12d.png)

Reply:
We'll have to push this off to a separate issue, I agree this is a good idea but I'm worried about scope creep
This PR has flagged up a number of things as we go through the docs with a fine-toothed comb

Reply:
Agreed.

Have opened https://github.com/mantidproject/mantid/issues/33543 to catch the requirement.